### DISCUSSION TITLE
Not sure where to post this.

### DISCUSSION THREAD

**Discussion by cooper2751** [lbbrhzn/ocpp#693] [General]
As subject says not sure where to post this but I just wanted to say I've been using this add-on to control my project EV charger for couple of months now and it's been flawless. Model EVA-07S-SE. Just incase it helps any one to know.

**Comment by lbbrhzn** [comment]
Thanks cooper! I'll add your device to the list of supported devices.

**Comment by tocklime** [comment]
@cooper2751 did you have to select the "Force Smart Charging feature profile" option to get it working? What controls do you have working? I'm just trying to set mine up. I don't seem to have control of the charging, but I think that might be because I left it in 'Eco+' mode before changing the ocpp server. Can't seem to change modes with this.
